### ISSUE TITLE
string pizzicato data missing in MIDI export

### ISSUE THREAD

**Issue by ApianbelleDev** [musescore/MuseScore#17172]
### Issue type

Other type of issue

### Bug description

when exporting to Midi, pizzicato data for string instruments(tested with viola, cello, and bass) is missing. this can be seen when exporting to midi, them importing from midi. happens with both snap pizz. , and normal pizz.

### Steps to reproduce

1. click file, export, and export as midi
2. import the midi file into either midi software, or back into musescore
3. pizz data is missing.

### MuseScore Version

4.0.2

### Regression

I don't know

### Operating system

Windows 11

**Comment by ApianbelleDev** [comment]
of course, I could be completely misunderstanding how MIDI data works in this case. pizz data may not usually export.

**Comment by abariska** [comment]
If we talk about another software - Pizzicato is that kind of data which could be mapped in different sound modules/plugins to different midi parameter. So, there's no way to make it working for any software. Only case when it can be helpful inside MU world is an opening score in earlier MU version than score was crated in. But seems like it's an edge case. 
@bkunda please, confirm that this issue should be closed as "won't fix".

BTW In MU3 pizz data was not exported also, so this's not a regression.

**Comment by ApianbelleDev** [comment]
ah, that makes sense. I'll figure it out in my DAW then. thanks!
